src/Models/WebMapServiceCatalogGroup.js

```diff
@@ -59,6 +59,12 @@ var WebMapServiceCatalogGroup = function(application) {
      */
     this.blacklist = undefined;
 
+    /**
+     * Specifies whether to use each WMS layer's "title" (default), "name", or "abstract" as the primary title.
+     * @type {String}
+     */
+    this.titlefield = 'title';
+
     knockout.track(this, ['url', 'dataCustodian', 'parameters', 'blacklist']);
```

Comment:
Yes, all public, writable properties should be observable, generally.  Sometimes you can make the case that it's not really useful.  The `preserveOrder` property in your other pull request, for example, only has an effect at a very specific time, so being notified when it changes doesn't seem very useful.  It doesn't really ever hurt to make something observable though.